app/src/main/java/org/kiwix/kiwixmobile/search/SearchActivity.java

```diff
@@ -12,6 +12,7 @@
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.widget.SearchView;
 import android.support.v7.widget.Toolbar;
+import android.util.Log;
```

Comment:
Probably you used Log while debugging the code. Now if there isn't any need for it then please remove this import.